exasol/slc_ci_setup/templates/slc_ci_check_for_build.yml.tmpl

```diff
@@ -0,0 +1,44 @@
+{% include "heading.tmpl" %}
+{% raw %}
+name: SLC-CI-Check-For-Build
+
+on:
+  workflow_call:
+    inputs:
+      flavor:
+          description: 'A flavor'
+          required: true
+          type: string
+    outputs:
+      continue:
+        description: "Indicates if need to build and test SLC"
+        value: ${{ jobs.check_for_build.outputs.continue }}
+
+jobs:
+  check_for_build:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Test branches
+        run: |
+            git checkout main
```

Comment:
git switch is recommended instead of checkout, however it might complain if the branch does not exist